imports/gridControl/GridControl.js

```diff
@@ -0,0 +1,378 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Toggle from 'material-ui/Toggle';
+import Slider from 'material-ui/Slider';
+import SelectField from 'material-ui/SelectField';
+import TextField from 'material-ui/TextField';
+import DropDownMenu from 'material-ui/DropDownMenu';
+import MenuItem from 'material-ui/MenuItem';
+import NumericInput from 'react-numeric-input';
+import actions from './actions';
+
+// import NumericInput from 'react-numeric-input';
+// import Slider from 'material-ui/Slider';
+const numericInputStyle = {
+  wrap: { height: '30px', width: '50px' },
+  input: { height: '30px', width: '50px' },
+};
+class GridControl extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+  }
+  componentWillReceiveProps = (nextProps) => {
+    const option = nextProps.option;
+    if (option) {
+      this.setState({ option });
```

Comment:
@julieyao05 , @appolloford , 
this may trigger another render, so duplicate render method.